src/xcode/ENA/ENA/Source/Services/CCLService/CCLScenariosHelper.swift

```diff
@@ -0,0 +1,54 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+class CCLScenariosHelper {
+	
+	// MARK: - Init
+			
+	init(cclService: CCLServable, store: HealthCertificateStoring) {
+		self.cclService = cclService
+		self.store = store
+	}
+	
+	// MARK: - Private
+
+	private let cclService: CCLServable
+	private let store: HealthCertificateStoring
+
+	// MARK: - Internal
+
+	func showAdmissionScenarios(
```

Comment:
There's nothing shown here, is there? I guess this is something like `selectValueViewModelForAdmissionScenarios`? And I think there is nothing async going on here, so you could just `return` the result

Reply:
done